Fss.Core/Types/MasterTypes.fs

```diff
@@ -1,6 +1,8 @@
 namespace Fss
 namespace Fss.Types
 
+open Fss.Types
```

Comment:
This opening statement is not needed I think

Reply:
Sorry, didn't noticed it. Fixed.